src/main/resources/project/ec_setup.pl

```diff
@@ -75,18 +75,18 @@
 
 # Data that drives the create step picker registration for this plugin.
 my %decommissionEnvironments = (
-	label     => "EF-Utilities - Decommission Environments",
-	procedure => "Decommission Environments",
+	label       => "EF-Utilities - Decommission Environments",
+	procedure   => "Decommission Environments",
 	description => "Decommission dynamic environments provisioned from pipeline, applications process or procedure",
-	category => "Deploy"
+	category    => "Deploy"
 );
 
-#my %createScheduleToTriggerGate = (
-#  label       => "EF-Utilities - createScheduleToTriggerGate",
-#  procedure   => "createScheduleToTriggerGate",
-#  description => "Create a schedule that trigger a gate at a particular time and date",
-#  category    => "Deploy"
-#);
+my %createScheduleToTriggerGate = (
+  label       => "EF-Utilities - createScheduleToTriggerGate",
```

Comment:
Label for procedure should contain short description of what it's doing. Without 'EC-<Plugin Name> -' prefix. For this case, i think, 'Create Schedule To Trigger Gate' or something similar would be enough.